lib/sportradar/api/nfl/stat_pack.rb

```diff
@@ -0,0 +1,30 @@
+module Sportradar
+  module Api
+    class Nfl::StatPack < Data
+      attr_accessor :response, :player, :name, :id, :position, :yards
+
+      def initialize(data)
+        @response = data
+        set_stats
```

Comment:
I wonder if we should define a method  for `set_stats` on this superclass statpack model; that raises an error, suggesting that it ought to be subclassed, not called directly. 

I did something similar here: https://github.com/beneggett/sportradar-api/blob/master/lib/sportradar/api/request.rb#L57-L59